### PR TITLE
Handles empty globes properly

### LINKED ISSUE #296: if globs aren't set then the rosapi's services aren't fully functional
If the globs for services and topics aren't set, then the following services in rosapi return an empty list:
```
/rosapi/services
/rosapi/topics
/rosapi/action_servers
/rosapi/get_services_for_type
/rosapi/service_node
...
```
and probably more

### PR DESCRIPTION
**Pull request by Behery**
The api is now able to handle empty globs. Meaning that the service calls are now functional even if all the topics/services/params are exposed. Fixes #296

### CODE REVIEW

File: rosapi/src/rosapi/proxy.py [hunk 1]
```diff
@@ -150,21 +159,31 @@ def filter_action_servers(topics):
...
+            if(possible_action_server == namespace and topic == "cancel"):
+                possibility[0] = 1
+            if(possible_action_server == namespace and topic == "feedback"):
+                possibility[1] = 1
+            if(possible_action_server == namespace and topic == "goal"):
+                possibility[2] = 1
+            if(possible_action_server == namespace and topic == "result"):
+                possibility[3] = 1
+            if(possible_action_server == namespace and topic == "status"):
+                possibility[4] = 1
```
Comment (jihoonl) [inline rosapi/src/rosapi/proxy.py]: Thanks for fixing this method too.
`filter_action_servers` method can be simplified like this way. Could you make this change?

```
    possibility = [0, 0, 0, 0, 0]
    action_topics = ['cancel', 'feedback', 'goal', 'result', 'status']
    for topic in sorted(topics):
        if (len(topic.split('/')) == 3):
            [empty, namespace, topic] = topic.split('/')
            if(possible_action_server != namespace):
                possible_action_server = namespace
                possibility = [0, 0, 0, 0, 0]
            if possible_action_server == namespace and topic in action_topics:
                possibility[action_topics.index(topic)] = 1
        if all(p == 1 for p in possibility):
            action_servers.append(possible_action_server)
```

File: rosapi/src/rosapi/proxy.py [hunk 2]
```diff
@@ -45,16 +45,20 @@
 
 from rosapi.msg import TypeDef
 
+from glob_helper import filter_globs, any_match
```
Comment (jihoonl) [inline rosapi/src/rosapi/proxy.py]: Use `from .glob_helper import filter_globs, any_match` to import from local module.

**Comment by Behery** [comment]
Hi @jihoonl could you please check the latest updates?
Thanks

**Comment by jihoonl** [comment]
Thanks.

**Pull request merged**